mappings/net/minecraft/screen/ScreenHandlerListener.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_1712 net/minecraft/screen/ScreenHandlerListener
```

Comment:
Maybe just "screen listener"?

Reply:
Hm, what exactly does this listen for

Reply:
or `ScreenUpdateListener`? `HandlerListener` sounds... extraneous?

Reply:
`ScreenHandleCallback` as this is like a callback when the screen handling is done